tcptracer-bpf.c

```diff
@@ -821,6 +841,48 @@ int kretprobe__inet_csk_accept(struct pt_regs *ctx)
 	return 0;
 }
 
+SEC("kprobe/fd_install")
+int kprobe__fd_install(struct pt_regs *ctx)
+{
+	u64 pid = bpf_get_current_pid_tgid();
+	u32 tgid = pid >> 32;
+	unsigned long fd = (unsigned long) PT_REGS_PARM1(ctx);
```

Comment:
I think the complexity and the time we'll spend extracting that from bcc and applying it here is not worth it.

In the end, the rewriting is done to make creating bpf programs easier and reduce verifier errors, but we already did the effort of writing pure bpf and making it compliant.